src/concourse/pipelines/infrastructure/ocw_studio_db_replication/pipeline.py

```diff
@@ -0,0 +1,84 @@
+from concourse.lib.constants import REGISTRY_IMAGE
+from concourse.lib.models.pipeline import (
+    AnonymousResource,
+    Command,
+    Identifier,
+    Input,
+    Job,
+    Output,
+    Pipeline,
+    Platform,
+    RegistryImage,
+    TaskConfig,
+    TaskStep,
+)
+
+postgres_resource = AnonymousResource(
+    type=REGISTRY_IMAGE, source=RegistryImage(repository="postgres", tag="12")
+)
+
+
+def db_replication_pipeline() -> Pipeline:
+    db_replication_job = Job(
+        name="restore-db",
+        build_log_retention={"builds": 10},
+        plan=[
+            TaskStep(
+                task=Identifier("run-pg-dump"),
+                privileged=False,
+                config=TaskConfig(
+                    platform=Platform.linux,
+                    image_resource=postgres_resource,
+                    outputs=[Output(name="dump")],
+                    params={
+                        "PGHOST": "((source.host))",
+                        "PGPORT": "((source.port))",
+                        "PGUSER": "((source.username))",
+                        "PGPASSWORD": "((source.password))",
+                    },
+                    run=Command(
+                        path="/bin/sh",
+                        args=[
+                            "-c",
+                            "pg_dump -Fc -d ((source.database)) > ./dump/db.dump",
```

Comment:
```suggestion
                            "pg_dump -v -Fc -d ((source.database)) > ./dump/db.dump",
```

Reply:
I checked this output with adding -v (verbose), it doesn't dump everything onto the console, just table + index names, which I think is a good compromise between "nothing" and "everything".